tests/installation/ipxe_install.pm

```diff
@@ -70,13 +70,23 @@ sub set_bootscript {
     my $arch = get_required_var('ARCH');
     my $autoyast = get_var('AUTOYAST', '');
     my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_hostname = get_var('OPENQA_HOSTNAME', 'openqa.suse.de');
 
     # trim all strings from variables to get rid of bogus whitespaces
     $arch =~ s/^\s+|\s+$//g;
     $autoyast =~ s/^\s+|\s+$//g;
     $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_hostname =~ s/^\s+|\s+$//g;
 
+    # For pxe/ipxe agama installation,we don't have linuxrc 'install=' parameter
+    # so we need to pass iso image to boot
     my $install = "install=$mirror_http";
+    if (is_agama) {
```

Comment:
I really wish we wouldn't bloat the set_bootscript functions further.. Maybe you could create a set_bootscript_agama() and add everything you need there? so we would just call the proper set_bootscript() in a conditional and not make the existing functions harder to read.

Reply:
Let me do that, thanks